zeppelin/zeppelin.rst

```diff
@@ -0,0 +1,19 @@
+===============
+Apache Zeppelin
```

Comment:
Perhaps it would be preferable to stick to the current standard of omitting 'Apache'.